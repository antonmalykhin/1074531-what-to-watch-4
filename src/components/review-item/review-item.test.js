import React from 'react';
import renderer from 'react-test-renderer';
import ReviewItem from './review-item.jsx';

const comment = {
  comment: `This movie is perfect in all its categories: credits, sound track, production, casting, writing, photography, editing, acting, and direction.
  I was amazed with the freedom of the use of the camera. This movie will change the way movies are made. Slow-mo, stills, black and white, and color were all used to brilliant effect.`,
  date: `2020-06-29T16:06:01.831Z`,
  rating: 5.6,
  user: {
    name: `Mollie`
  }
};

it(`ReviewItem render correctly`, () => {
  const tree = renderer.create(
      <ReviewItem review={comment} />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
