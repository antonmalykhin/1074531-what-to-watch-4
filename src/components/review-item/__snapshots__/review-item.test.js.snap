// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewItem render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      This movie is perfect in all its categories: credits, sound track, production, casting, writing, photography, editing, acting, and direction.
  I was amazed with the freedom of the use of the camera. This movie will change the way movies are made. Slow-mo, stills, black and white, and color were all used to brilliant effect.
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Mollie
      </cite>
      <time
        className="review__date"
        dateTime="2016-12-20"
      >
        June 29, 2020
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    5.6
  </div>
</div>
`;
